systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -148,21 +143,21 @@ void testMirrorMaker2(ExtensionContext extensionContext) {
         String kafkaPodJson = TestUtils.toJsonString(kubeClient().getPod(testStorage.getNamespaceName(), podName));
 
         assertThat(kafkaPodJson, hasJsonPath(StUtils.globalVariableJsonPathBuilder(0, "KAFKA_CONNECT_BOOTSTRAP_SERVERS"),
-                hasItem(KafkaResources.plainBootstrapAddress(kafkaClusterTargetName))));
+                hasItem(KafkaResources.plainBootstrapAddress(testStorage.getTargetClusterName()))));
         assertThat(StUtils.getPropertiesFromJson(0, kafkaPodJson, "KAFKA_CONNECT_CONFIGURATION"), is(expectedConfig));
         testDockerImagesForKafkaMirrorMaker2(testStorage.getClusterName(), clusterOperator.getDeploymentNamespace(), testStorage.getNamespaceName());
 
         verifyLabelsOnPods(testStorage.getNamespaceName(), testStorage.getClusterName(), "mirrormaker2", KafkaMirrorMaker2.RESOURCE_KIND);
         verifyLabelsForService(testStorage.getNamespaceName(), testStorage.getClusterName(), "mirrormaker2", "mirrormaker2-api", KafkaMirrorMaker2.RESOURCE_KIND);
-        verifyLabelsForConfigMaps(testStorage.getNamespaceName(), kafkaClusterSourceName, null, kafkaClusterTargetName);
-        verifyLabelsForServiceAccounts(testStorage.getNamespaceName(), kafkaClusterSourceName, null);
+        verifyLabelsForConfigMaps(testStorage.getNamespaceName(), testStorage.getSourceClusterName(), null, testStorage.getTargetClusterName());
+        verifyLabelsForServiceAccounts(testStorage.getNamespaceName(), testStorage.getSourceClusterName(), null);
 
         LOGGER.info("Now setting Topic to {} and cluster to {} - the messages should be mirrored",
-            sourceMirroredTopicName, kafkaClusterTargetName);
+            testStorage.getSourceTopicName(), testStorage.getTargetClusterName());
 
         clients = new KafkaClientsBuilder(clients)
-            .withTopicName(sourceMirroredTopicName)
-            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(kafkaClusterTargetName))
+            .withTopicName(testStorage.getSourceTopicName())
```

Comment:
This will not work FMPOV.
The name of the mirrored topic in target cluster follows this pattern - `<source_cluster_name>.<topic_name>` and you are using here `<topic_name>-source`.

Yeah there is one or two testcases were we are using just source topic name, as we have a special configuration there (the `IdentityReplicationPolicy`), but this is not the case for all other testcases.